src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -150,7 +150,7 @@ std::vector<std::string> disabledTestPatterns() {
         R"(.*smoke_Proposal_(Static|Dynamic)_Test_Case1/ProposalLayerCPUTest.*)",
         // Issue: 111418
         R"(.*smoke_Snippets_ConvertStub/ConvertStub\.CompareWithRefImpl/IS.*_OT=\(bf16\)_#N=2_#S=2_targetDevice=CPU.*)",
-        R"(.*smoke_Snippets_Convert/Convert\.CompareWithRefImpl/IS.*_IT=\(f32\)_OT=\(u8\)_#N=1_#S=1_targetDevice=CPU.*)",
+        R"(.*smoke_Snippets_Convert/Convert\.CompareWithRefImpl/IS.*_IT=\((f32|f16)\)_OT=\(u8\)_#N=1_#S=1_targetDevice=CPU.*)",
```

Comment:
Could you please elaborate why you had to add `f16` to skip_test_config for all platforms?

Reply:
Thanks Alexandra for the comment! Following is the values I printed out, when I was debugging this issue.
The issue is not on input precision to be f32 or f16, it's on output precision to be u8. As we can see, in reference ouput, all the negative values are converted to be 0. The reference results are incorrectly because the ConvertTruncation node is explicitly inserted in the reference model https://github.com/openvinotoolkit/openvino/blob/master/src/tests/ov_helpers/ov_snippets_models/src/subgraph_convert.cpp#L29, they should be the negative values as in target output.
As this issue is already tracked by ticket 111418, so I merged skipping test cases with output u8 to this ticket. Please feel free to have further discussions.


[ RUN      ] smoke_Snippets_Convert/Convert.CompareWithRefImpl/IS[0]=[]_TS[0]=(2.16)_IT=(f16)_OT=(u8)_#N=1_#S=1_targetDevice=CPU

MEM_USAGE=142132KB
input: 5 6.80078 6.19922 4.80078 9.79688 0.399902 7 7.19922 -0.600098 _`-8.39844`_ -5.80078 3.19922 0.199951 7.60156 -5.80078 3.19922 -6.19922 0.399902 -4.19922 -5.60156 -7 4.39844 -5.80078 9.60156 -7 2.80078 -9.79688 -8.79688 -7 0.399902 6.19922 -5.60156 
reference-output: 5 6 6 4 9 0 7 7 0 _`0`_ 0 3 0 7 0 3 0 0 0 0 0 4 0 9 0 2 0 0 0 0 6 0 
target-output: 5 6 6 4 9 0 7 7 0 _`248`_ 251 3 0 7 251 3 250 0 252 251 249 4 251 9 249 2 247 248 249 0 6 251